Scrape-Subreddit-Users-master/GetSubredditSubscribersComments.py

```diff
@@ -6,10 +6,11 @@
 #    entered in the dict
 
 import database as db
-from GetSubredditSubscribers import *
+from GetSubredditSubscribers import * # Let's talk about this -- RMD
```

Comment:
So boy howdy, `from foo import *` is dangerous waters. Unless you've been very careful to make sure you know precisely what you're importing, you can wind up with all sort of weird side effects.